apps/files_sharing/lib/Controller/ShareAPIController.php

```diff
@@ -197,6 +199,22 @@ protected function formatShare(IShare $share, ?Node $recipientNode = null): arra
 		$result['item_size'] = $node->getSize();
 		$result['item_mtime'] = $node->getMTime();
 
+		if ($this->trustedServers !== null && in_array($share->getShareType(), [IShare::TYPE_REMOTE, IShare::TYPE_REMOTE_GROUP])) {
+			$result['is_trusted_server'] = false;
+			$sharedWith = $share->getSharedWith();
+			$remoteIdentifier = is_string($sharedWith) ? strrchr($sharedWith, '@') : false;
+			if ($remoteIdentifier !== false) {
+				$remote = substr($remoteIdentifier, 1);
+				try {
+					if ($this->trustedServers->isTrustedServer($remote)) {
+						$result['is_trusted_server'] = true;
+					}
```

Comment:
```suggestion
				$result['is_trusted_server'] = $this->trustedServers->isTrustedServer($remote);
```

nitpick, we are good to go with the current version.